django-stubs/contrib/admin/widgets.pyi

```diff
@@ -21,12 +21,24 @@ class FilteredSelectMultiple(forms.SelectMultiple):
         choices: _Choices = ...,
     ) -> None: ...
 
-class AdminDateWidget(forms.DateInput):
+class BaseAdminDateWidget(forms.DateInput):
+    class Media:
```

Comment:
CC @flaeppe 

I think that defining such classes may cause an error when defining subclasses of widgets.

It is close to out `Meta` case.

Reply:
@sobolevn I have some sympathy for what you said. In the case of this part meta, I will delete it and commit it :)

Reply:
Let's ignore it for now, we can later decide what to do.

Reply:
I revised what you said :)